data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -318,6 +318,52 @@ then
     record_service_stage_success
 fi
 
+if [ ! -f cmdbootline-nto-bootstrap.done ]; then
+	record_service_stage_start "cmdbootline-nto-bootstrap"
+
+	rm --recursive --force cmdbootline-nto-bootstrap
+
+	# Create folders for overlay and podman command to use them
+	tmpfolder=$(mktemp -d)
+	echo ${tmpfolder}
+	podmancmd="bootkube_podman_run -d --rm "
+	for folder in var/lib/tuned/profile-data etc/modprobe.d etc/sysconfig etc/kubernetes etc/sysctl.d etc/systemd run sys lib/modules var/lib/kubelet usr/lib etc/tuned; do
+		lowerfolder="/${folder}"
+		if [ ! -d "$lowerfolder" ]; then
+			lowerfolder="${tmpfolder}/${folder}/lower"
+			mkdir -p ${lowerfolder}
+		fi
+		upperfolder="${tmpfolder}/${folder}/upper"
+		workfolder="${tmpfolder}/${folder}/work"
+		mkdir -p ${upperfolder}
+		mkdir -p ${workfolder}
+		podmancmd="${podmancmd} -v ${lowerfolder}:/${folder}:O,upperdir=${upperfolder},workdir=${workfolder}"
+	done
+
+	podmancmd="${podmancmd} --name tuned --entrypoint \"cluster-node-tuning-operator\" ${NODE_TUNING_OPERATOR_IMAGE} render-bootcmd-mc --asset-input-dir=/assets/manifests --asset-output-dir=/assets/cmdbootline-nto-bootstrap"
```

Comment:
```suggestion
	podmancmd="${podmancmd} --name tuned ${NODE_TUNING_OPERATOR_IMAGE} render-bootcmd-mc --asset-input-dir=/assets/manifests --asset-output-dir=/assets/cmdbootline-nto-bootstrap"
```
we don't have to specify the entrypoint here, because `cluster-node-tuning-operator` is the default entrypoint specified in the Dockerfile build.
if you still want to explicitly set it, a full path is needed:
```bash
--entrypoint /usr/bin/cluster-node-tuning-operator
``` 
otherwise the container won't run with this error:
```bash
Error: runc: runc create failed: unable to start container process: exec: "\"cluster-node-tuning-operator\"": executable file not found in $PATH: OCI runtime attempted to invoke a command that was not found
```

Reply:
Yes, you are right about not needing to explicitly designate the entrypoint as it is the default one but running this same command myself and got a proper output

```
podmancmd: bootkube_podman_run  --rm  --name tuned --entrypoint "cluster-node-tuning-operator" quay.io/jojosneg/origin-cluster-node-tuning-operator:master render -h                                                                                                                     
Render performance-addon-operator manifests

Usage:
  cluster-node-tuning-operator render [flags]

Flags:
      --add_dir_header                   If true, adds the file directory to the header
      --alsologtostderr                  log to standard error as well as files
      --asset-input-dir string           Input path for the assets directory. (Can be a comma separated list of directories.) (default "/assets")                                                                                                                                        
      --asset-output-dir string          Output path for the rendered manifests.
  -h, --help                             help for render
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --log_file string                  If non-empty, use this log file
      --log_file_max_size uint           Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited. (default 1800)                                                                                                       
      --logtostderr                      log to standard error instead of files (default true)
      --skip_headers                     If true, avoid header prefixes in the log messages
      --skip_log_headers                 If true, avoid headers when opening log files
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          number for the log level verbosity
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

Global Flags:
      --enable-leader-election   Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager. (default true)                                                                                                                    
      --kubeconfig string        Paths to a kubeconfig. Only required if out-of-cluster.
      --one_output               If true, only write logs to their native severity level (vs also writing to each lower severity level; no effect when -logtostderr=true)                                                                                                                
      --version                  Show program version and exit.
```

Reply:
The problem happens when I do not use `eval` to run the command.
Anyhow as it is the default entrypoint let's delete it.